app-arch/bzip2/bzip2-9999.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/hanetzer/bzip2"
+	inherit autotools git-r3
+else
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+	SRC_URI="mirror://gentoo/${P}.tar.gz"
+fi
+
+LICENSE="BZIP2"
+SLOT="0/1" # subslot = SONAME
+IUSE="static static-libs"
+
+DOCS=( manual.pdf NEWS README.autotools README README.XML.STUFF )
+HTML_DOCS=( manual.html )
+
+src_prepare() {
+	default
+	if [[ ${PV} == *9999* ]]; then
+		eautoreconf
+	fi
+}
+
+multilib_src_configure() {
+	# use static && append-ldflags -static # doesn't seem to work...
+	# fails at gen_usr_ldscript because adding -static to ldflags
+	# prevents generation of libbz2.so
+
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static)
+}
+
+multilib_src_install() {
+	emake DESTDIR="${E}" install
```

Comment:
What's just "E"? Where does this coming from? Cannot find it in any multilib eclass.

Reply:
Yeah, just "D" would be OK in this phase. I picked "ED" because it was used later in the same function to match style. But yes, final solution should only use D.